src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -180,6 +186,11 @@ case class SeqIndexNegativeError(node: Source.Verifier.Info, index: Source.Verif
   override def message: String = s"Index ${index.origin.tag.trim} into ${node.origin.tag.trim} might be negative"
 }
 
+case class OverflowErrorReason(node: Source.Verifier.Info) extends VerificationErrorReason {
+  override def id: String = "integer_overflow_error"
+  override def message: String = s"Expression ${node.origin.tag} may cause integer overflow"
+}
+
```

Comment:
Is this actually required. I would assume that OverflowError suffices.

Reply:
I would argue that it is needed in order to have meaningful error messages. If this class was not here, we would most likely use the `AssertionFalseError` which would lead to messages such as `Assertion X might not hold`, where X is the condition that causes the overflow error